networking/routes/route-configuration.adoc

```diff
@@ -66,3 +66,8 @@ include::modules/nw-ingress-edge-route-default-certificate.adoc[leveloffset=+1]
 include::modules/nw-ingress-reencrypt-route-custom-cert.adoc[leveloffset=+1]
 
 include::modules/nw-router-configuring-dual-stack.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```

Comment:
Add'l resources are no longer supposed to have sentences per Jupiter compatibility
```suggestion
* xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option]
```